packages/imperative/src/config/src/api/ConfigLayers.ts

```diff
@@ -49,11 +51,19 @@ export class ConfigLayers extends ConfigApi {
                 layer.properties = JSONC.parse(fileContents.toString()) as any;
                 layer.exists = true;
             } catch (e) {
-                throw new ImperativeError({
-                    msg: `Error parsing JSON in the file '${layer.path}'.\n` +
-                        `Please check this configuration file for errors.\nError details: ${e.message}\nLine ${e.line}, Column ${e.column}`,
-                    suppressDump: true
-                });
+                const msg = `Error parsing JSON in the file '${layer.path}'.\n` +
+                `Please check this configuration file for errors.\nError details: ${e.message}\nLine ${e.line}, Column ${e.column}`;
+                if (!opts.ignoreErrors){
```

Comment:
Curious if we should  provide a default value in the function signature (to `false` based on this implementation) 😋

Reply:
rather than setting default here what about `ignoreErrors: ignoreErrors || false` on line 182 in `packages/imperative/src/config/src/Config.ts`

Reply:
now that im thinking about that retroactively, this will cause ignoreErrors to always have a value and im not sure if that's wanted behavior given it was defined to be optional